authentication-service/src/main/kotlin/com/saveourtool/save/authservice/security/ConvertingAuthenticationManager.kt

```diff
@@ -22,44 +22,37 @@ import reactor.kotlin.core.publisher.switchIfEmpty
  */
 @Component
 class ConvertingAuthenticationManager(
-    private val authenticationUserDetailsService: AuthenticationUserDetailsService
+    private val authenticationUserRepository: AuthenticationUserRepository,
 ) : ReactiveAuthenticationManager {
     /**
-     * Authenticate user, by checking the received data, which converted into UsernamePasswordAuthenticationToken
-     * by [CustomAuthenticationBasicConverter] with record in DB
+     * Authenticate user, by checking the received data
      *
      * @return augmented mono of UsernamePasswordAuthenticationToken with additional details
      * @throws BadCredentialsException in case of bad credentials
      */
     override fun authenticate(authentication: Authentication): Mono<Authentication> = if (authentication is UsernamePasswordAuthenticationToken) {
-        val (name, identitySource) = authentication.extractUserNameAndIdentitySource()
-        authenticationUserDetailsService.findByUsername(name)
-            .cast<IdentitySourceAwareUserDetails>()
-            .filter {
-                it.identitySource == identitySource
-            }
+        val name = authentication.username()
+        blockingToMono {
+            authenticationUserRepository.findByName(name)?.toAuthenticationWithDetails()
+        }
             .switchIfEmpty {
                 Mono.error { BadCredentialsException(name) }
             }
             .onErrorMap {
                 BadCredentialsException(name)
             }
-            .map {
-                it.toAuthenticationWithDetails(authentication)
-            }
     } else {
         Mono.error { BadCredentialsException("Unsupported authentication type ${authentication::class}") }
     }
 
-    private fun IdentitySourceAwareUserDetails.toAuthenticationWithDetails(authentication: Authentication) =
+    private fun User.toAuthenticationWithDetails(): Authentication =
```

Comment:
we will need to update SAVE API? Now password is now set with ":"

Reply:
it's already updated. need to test

Reply:
will do it as part of #2336